releasenotes/notes/add-opflow-is-hermitian-method-6a461549e3c6b32c.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Method :meth:`~qiskit.opflow.OperatorBase.is_hermitian` is added to check whether
+    the operator is Hermite or not. :class:`~qiskit.algorithms.NumPyEigensolver` and
+    :class:`~qiskit.algorithms.NumPyMinimumEigensolver` use eigh or eigsh to solve the
+    eigenvalue problem when the operator is hermitian.
```

Comment:
This may actually be two features - one is adding the method, and the other is the change to the eigensolvers.  The first line should mention that the method is added to `OperatorBase` - the `~` in the crossref will cause only `is_hermitian` to be shown, so it won't be clear what object it's added to.